seqr/views/utils/export_utils_2_3_tests.py

```diff
@@ -14,36 +16,37 @@ class ExportTableUtilsTest(TestCase):
 
     def test_export_table(self):
         header = ['column1', 'column2']
-        rows = [['row1_v1', 'row1_v2'], ['row2_v1', 'row2_v2']]
+        rows = [['row1_v1\xe2', 'row1_v2'], ['row2_v1', 'row2_v2']]
 
         # test tsv format
         response = export_table('test_file', header, rows, file_format='tsv')
 
         self.assertEqual(response.status_code, 200)
         self.assertEqual(response.get('content-disposition'), 'attachment; filename="test_file.tsv"')
-        self.assertEqual(response.content, '\n'.join(['\t'.join(row) for row in [header]+rows]) + '\n')
+        self.assertEqual(response.content.decode('utf-8', errors = 'ignore'),
+                         '\n'.join(['\t'.join(row) for row in [header]+rows]) + '\n')
 
         # test Excel format
         response = export_table('test_file', header, rows, file_format='xls')
         self.assertEqual(response.status_code, 200)
         self.assertEqual(response.get('content-disposition'), 'attachment; filename="test_file.xlsx"')
-        wb = load_workbook(StringIO(response.content))
+        wb = load_workbook(BytesIO(response.content))
         worksheet = wb.active
 
-        self.assertListEqual([cell.value for cell in worksheet['A']], ['Column1', 'row1_v1', 'row2_v1'])
+        self.assertListEqual([cell.value for cell in worksheet['A']], ['Column1', 'row1_v1\xe2', 'row2_v1'])
         self.assertListEqual([cell.value for cell in worksheet['B']], ['Column2', 'row1_v2', 'row2_v2'])
         self.assertEqual([cell.value for cell in worksheet['C']], [None, None, None])
 
         # test unknown format
-        self.assertRaisesRegexp(ValueError, '.*format.*',
+        self.assertRaisesRegex(ValueError, '.*format.*',
             lambda: export_table('test_file', header, rows, file_format='unknown_format')
         )
 
     @mock.patch('seqr.views.utils.export_utils.zipfile.ZipFile')
     def test_export_multiple_files(self, mock_zip):
         mock_zip_content = {}
         def mock_write_zip(file, content):
-            mock_zip_content[file] = unicode(content)
```

Comment:
This is a crucial line of test code that is making sure this line is working - https://github.com/macarthur-lab/seqr/pull/1347/files#diff-f83ee929ae3748972d14c31ecc67bf2dR86

Since everything is encoded before write zip is called, this call should always run without error, and thats what this is testing. Taking this out means that encoding is not properly tested, which means a real life bug could go through

Reply:
According to the online [document](https://docs.python.org/3/library/zipfile.html#zipfile.ZipFile.writestr)
```
ZipFile.writestr(zinfo_or_arcname, data, compress_type=None, compresslevel=None)

> Write a file into the archive. The contents is data, which may be either a str or a bytes instance; 
> if it is a str, it is encoded as UTF-8 first.
```
Explicitly encode the data is optional. It will be more concise if write `str` rather than encode them to `bytes`.

Reply:
As the discussion on today's meeting, `unicode(content)` will be changed to `str(content, 'ascii', errors='ignore') to strip out all unicode chars.